src/components/EventCard.vue

```diff
@@ -1,26 +1,122 @@
 <template>
-  <q-card class="event-card" flat>
-    <img class="event-card__image" :src="event.imageUrl" />
-
-    <q-card-section class="event-card__description">
-      <div class="event-card__description-title">{{event.title}}</div>
-      <div class="event-card__description-date q-mb-sm">{{event.date}}</div>
-      <div class="event-card__description-text">{{event.descriptionText}}</div>
-      <div class="event-card__location row items-center q-mt-md">
-        <q-icon name="location_on" class="event-card__location-icon"></q-icon>
-        <p class="event-card__location-address">{{event.location}}</p>
+  <router-link :to="`/eventos/${ event.id }`">
+    <q-card class="event-card bg-secondary q-mr-md" :style="isFullWidth" flat>
+      <div class="relative-position">
+        <div class="event-card__subjects">
+          
+        </div>
+        <div class="event-card__image full-width" :style="eventImage"/>
       </div>
-    </q-card-section>
-  </q-card>
+      
+      <q-card-section class="event-card__description">
+        <div class="event-card__description-title text-bold">{{ event.name }}</div>
+        <div class="event-card__description-date q-mb-sm text-uppercase text-medium-purple text-bold">
+          {{ eventDate }} {{ eventStartTime }}
+        </div>
+        <div class="event-card__description-text ellipsis-3-lines text-medium-gray">{{ event.description }}</div>
+        <div class="event-card__location row items-center q-mt-md">
+          <q-icon name="location_on" color="strong-pink" class="event-card__location-icon"></q-icon>
+          <p class="event-card__location-address ellipsis q-ml-sm q-mb-none text-darker-purple">
+            {{ event.place }}
+          </p>
+        </div>
+      </q-card-section>
+    </q-card>
+  </router-link>
 </template>
 
 <script>
+import { formatDate, formatTime } from 'src/helpers/filters'
+
 export default {
+  data () {
+    return {
+      windowSize: ''
+    }
+  },
+
   props: {
     event: {
       type: Object,
       required: true
     }
+  },
+
+  created () {
+    this.windowSize = window.innerWidth
+    window.addEventListener('resize', this.setWindowSize)
+  },
+
+  computed: {
+    eventImage () {
+      return `background-image: url(${this.event?.avatar?.medium});`
+    },
+    
+    eventDate () {
+      return formatDate(this.event.date)
+    },
+
+    eventStartTime () {
+      return formatTime(this.event.start_time)
+    },
+
+    isFullWidth () {
```

Comment:
- isFullWidth me parece que sempre vai retornar um boolean, melhor mudar para outra coisa, talvez só um `fullWidth` já resolva.
- cria uma computada isSubject que retorna this.$route.name === 'Subject'

```js
return this.windowSize < 750 && this.isSubject && 'width: 100%;'
```
assim n resolve?